reports/views.py

```diff
@@ -38,6 +41,17 @@ def naive_datetime_to_tz_aware(datetime: str) -> str:
     return datetime + "T00:00:00.000Z" if datetime else None
 
 
+class LogAccessMixin:
```

Comment:
What do you think, would this be worthy to add to django-auditlog-extra for additional DRF support?

Reply:
A view mixin is not a good solution at all, because it can only take care of the first level of the object. What if a detail view or a list view shows some embedded objects or object relations? A generic view mixin cannot know about those. We can provide a get_queryset method or get_context_data method through a mixin, but the context scope of it is still unknown.

A generic reusable mixin would make more sense if it would be on serialization level. Some kind of object serializer is called everytime, no matter how deep we go.  It's easier to provide a generic mixin in GraphQL scope, because there a get_node is called for every level of the response object.